docs/source/teams/roles_and_permissions.rst

```diff
@@ -96,6 +97,39 @@ and they may only be granted **Can view** access to datasets.
 Guests cannot create new datasets, clone existing datasets, export datasets, or
 view other users of the deployment.
 
+.. _teams-groups:
+
+Groups
+___________
+
+User groups in FiftyOne Teams allow organization admins to manage a collection 
+of users as a single entity. This simplifies the process of assigning 
+permissions to multiple users, making it more efficient to control access to 
+datasets.
+
+Admins can manage groups through the "Settings > Team > Groups" page. 
+Each group can be given specific dataset access permissions, which apply to 
+all users within the group (except for guests).
+
+.. image:: /images/teams/admin_team_groups_page.png
+   :alt: admin-team-groups-page
+   :align: center
+
+Admins can create a new group by clicking on "Create group" and then adding 
+existing users to the group by clicking on "Add users". 
+
+.. image:: /images/teams/admin_create_group.png
+   :alt: admin-create-group
+   :align: center
+
+.. image:: /images/teams/admin_add_users_to_team.png
+   :alt: admin-team-add-users-to-team
+   :align: center
+
+.. note::
+   Non-existing users cannot be directly added to a group. Users must be 
```

Comment:
nit: add empty line after `.. note::` to match formatting of other notes in the docs (not sure if this will render correctly as-is? sphinx can be weird sometimes)